### ISSUE TITLE
Advice for 'Warning: Potential Security Risk Ahead'

### ISSUE THREAD

**Issue by funkytwig** [nginx-proxy/docker-letsencrypt-nginx-proxy-companion#532]

**Comment by funkytwig** [comment]
Hi, first what a great container, thanks.
In the process of setting up sites sometimes initially 'CA marked some of the authorizations as invalid, which likely means it could not ...' but it seem to sort itself out quite quickly, some insight in this would be good.
However, my biggest problem is the certificates seem to be generated OK but then I get 'Warning: Potential Security Risk Ahead' and it won't even let me add an exception.  it can sort itself out but can take a bit of time.  It would be good if it could be set up so I can at least put in an exception when I am testing.
Basically, I am asking for advice on how to minimize these problems, maybe something that could go in the README.. Have seen some references to test certificates/servers but not sure what best practice is or exactly how to use them.
Thanks,
Ben

**Comment by buchdag** [comment]
Hi, sorry for the delay.

> In the process of setting up sites sometimes initially 'CA marked some of the authorizations as invalid, which likely means it could not ...' but it seem to sort itself out quite quickly, some insight in this would be good.

https://github.com/JrCs/docker-letsencrypt-nginx-proxy-companion/blob/master/docs/Invalid-authorizations.md

> However, my biggest problem is the certificates seem to be generated OK but then I get 'Warning: Potential Security Risk Ahead' and it won't even let me add an exception. it can sort itself out but can take a bit of time. 

Are you still experiencing this issue ? If yes please provide detailed info about your setup (full configuration, container logs, etc). It might have been linked to #543 which is now fixed.

> It would be good if it could be set up so I can at least put in an exception when I am testing.

The reason why you can't set an exception is probably because `nginx-proxy` uses HSTS by default. We can't do anything about that from this container end, and understanding how `nginx-proxy` works before using `letsencrypt-nginx-proxy-companion` is the user's responsibility.

**Comment by buchdag** [comment]
No answer from issue creator since june last year, closing.
